### PR TITLE
feat(PrivatePages): add header section template with common styles

### PR DESCRIPTION
**Pull request by fcoronelmakingsense**
- Create shared component with the internal header section styles and structure for reuse in internal pages.

image

### CODE REVIEW

File: src/components/NewFeatures/NewFeatures.js [hunk 1]
```diff
@@ -1,27 +1,23 @@
...
-              <h2>Funcionalidades Nuevas</h2>
-              <p>
-                Features sin acceso para los usuarios todavía y con trabajo pendiente de contenidos
-                y diseño.
-              </p>
-            </div>
-          </div>
-          <span className="arrow"></span>
+      <HeaderSection>
+        <div className="col-sm-12 col-md-12 col-lg-12">
```
Comment (andresmoschini) [inline src/components/NewFeatures/NewFeatures.js:13]: This div cannot be part of the HeaderSection?

Comment (fcoronelmakingsense) [reply src/components/NewFeatures/NewFeatures.js:13]: Because in some pages use two times, like in reports

Comment (andresmoschini) [reply src/components/NewFeatures/NewFeatures.js:13]: 👍 
In the future, we could parametrize it, something like this:

```html
       <HeaderSection col-sm=12 col-md=12 col-lg=12>
```

**Pull request merged**